### ISSUE TITLE
Dice so Nice problem

### ISSUE THREAD

**Issue by Stexinator** [witchertrpg-foundryvtt/TheWitcherTRPG#20]
There is a problem with the Dice so Nice module that when it is critical or fumble, the dices from the module does not appear, the result is printed directly in the chat, is there a possible resolution for this?

**Comment by ortegamarcel** [comment]
Yes. In my (old) fork, I already fixed it. I will apply that fix here as well.
